lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -4,38 +4,38 @@
 # @see Qa::Authorities::LinkedData::SearchConfig
 module Qa::Authorities
   module LinkedData
-    module TermConfig
-      # Return the full configuration for term fetch.
-      # @return [String] the term configuration
-      def term_config
-        cfg = config_value(authority_config, :term)
-        cfg = nil if cfg.is_a?(Hash) && cfg.empty?
-        cfg
+    class TermConfig
+      # @param [Hash] config the term portion of the config
+      def initialize(config)
+        @term_config = config
       end
 
+      attr_reader :term_config
+      private :term_config
+
       # Does this authority configuration have term defined?
       # @return [True|False] true if term fetching is configured; otherwise, false
       def supports_term?
-        !term_config.nil?
+        term_config.present?
       end
 
       # Return term url encoding defined in the configuration for this authority.
       # @return [Hash] the configured term url
       def term_url
-        config_value(term_config, :url)
+        Config.config_value(term_config, :url)
```

Comment:
Search was switched to use fetch instead of config_value.  To be consistent, term should be updated to use hash fetch too.